src/ActivityForm.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import "./ActivityForm.css";
+
+export default function ActivityForm({ onAddActivity }) {
+  //console.log(onAddActivity);
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const newActivity = {
+      name: event.target.elements.activity.value,
+      isForGoodWeather: event.target.elements.weatherActivity.checked,
+    };
+
+    onAddActivity(newActivity);
+    console.log("ActivityForm: ", newActivity);
+
+    event.target.reset();
+    event.target.elements.activity.focus();
+  }
+
+  return (
+    <div>
+      <h1>Add new Activity</h1>
+      <form onSubmit={handleSubmit}>
+        <label> Add New Activity: </label>
```

Comment:
Using a `<label> tag for e.g. a heading like _"Add New Activity"_ is not semantically correct.

```suggestion
        <h2> Add New Activity: </h2>
```